rfcs/text/0000/container.yml

```diff
@@ -0,0 +1,50 @@
+- name: container
+  fields:
+    - name: cpu.usage
+      type: scaled_float
+      scaling_factor: 1000
+      level: extended
+      short: Percent CPU used, between 0 and 1.
+      description: >
+        Percent CPU used which is normalized by the number of CPU cores and it
+        ranges from 0 to 1. Scaling factor: 1000.
+
+    - name: memory.usage
```

Comment:
Good question :) `memory.usage` maybe will eventually add into ECS. We didn't add it in the first place because most of the modules in Metricbeat collecting host metrics don't have access to memory usage.